test_scripts/Polices/External_UCS/External_Consent_OFF/008_ATF_Policies_External_Consent_OFF_disallowed_by_external_consent_entities_omitted_user_consent_allowed.lua

```diff
@@ -0,0 +1,151 @@
+  -- remove preload_pt from json file
+  local parent_item = {"policy_table","module_config"}
+  local removed_json_items = {"preloaded_pt"}
+  common_functions:RemoveItemsFromJsonFile("/tmp/ptu_update.json", parent_item, removed_json_items)
+  -- update policy table
+  common_functions_external_consent:UpdatePolicy(self, "/tmp/ptu_update.json")
+end
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- Check GetListOfPermissions response with empty externalConsentStatus array list. Get group id.
+--------------------------------------------------------------------------
+Test["TEST_NAME_OFF".."_Precondition_GetListOfPermissions"] = function(self)
+  --hmi side: sending SDL.GetListOfPermissions request to SDL
+  local request_id = self.hmiConnection:SendRequest("SDL.GetListOfPermissions")
+  -- hmi side: expect SDL.GetListOfPermissions response
+  EXPECT_HMIRESPONSE(request_id,{
+      result = {
+        code = 0,
+        method = "SDL.GetListOfPermissions",
+        allowedFunctions = {{name = "ConsentGroup001", allowed = nil}},
+        externalConsentStatus = {}
+      }
+    })
+  :Do(function(_,data)
+      id_group_1 = common_functions_external_consent:GetGroupId(data, "ConsentGroup001")
+    end)
+end
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- HMI sends OnAppPermissionConsent with External Consent status = OFF and consented function = allowed
+--------------------------------------------------------------------------
+Test["TEST_NAME_OFF" .. "_Precondition_HMI_sends_OnAppPermissionConsent"] = function(self)
+  local hmi_app_id_1 = common_functions:GetHmiAppId(config.application1.registerAppInterfaceParams.appName, self)
+  -- hmi side: sending SDL.OnAppPermissionConsent for applications
+  self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent", {
+      appID = hmi_app_id_1, source = "GUI",
+      externalConsentStatus = {{entityType = 2, entityID = 5, status = "OFF"}},
+      consentedFunctions = {{name = "ConsentGroup001", id = id_group_1, allowed = true}}
+    })
+  self.mobileSession:ExpectNotification("OnPermissionsChange")
+end
+
+--------------------------------------------------------------------------
+-- Main check:
+-- RPC is allowed to process.
+--------------------------------------------------------------------------
+Test["TEST_NAME_OFF" .. "_MainCheck_RPC_is_allowed"] = function(self)
+  --mobile side: send SubscribeWayPoints request
+  self.mobileSession:SendRPC("SubscribeWayPoints",{})
+  --hmi side: expected SubscribeWayPoints request
+  EXPECT_HMICALL("Navigation.SubscribeWayPoints")
+  :Do(function(_,data)
+      --hmi side: sending Navigation.SubscribeWayPoints response
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS",{})
+    end)
+  --mobile side: SubscribeWayPoints response
+  EXPECT_RESPONSE("SubscribeWayPoints", {success = true , resultCode = "SUCCESS"})
```

Comment:
@istoimenova, Please update with usage of _correlationId_ because current version can be unstable